governance/votes/Executive vote - August 22, 2024.md

```diff
@@ -0,0 +1,142 @@
+
+### Update GSM Delay
+
+- **Authorization**: [Governance Facilitator Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/3), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, the [GSM Pause Delay](https://mips.makerdao.com/mips/details/MIP113#10-1-gsm-governance-security-module-pause-delay) will be updated as follows:
+
+- The GSM Pause Delay will be increased by 14 hours from 16 hours to **30 hours**.
+
+### Update LitePSM Keeper Network Job
+
+- **Authorization**: [Ecosystem Approval](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839/2), [Poll 1128](https://vote.makerdao.com/polling/QmU7XJ6X)
+- **Proposal**: [Forum Post](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-2-major-migration-proposed-parameters/24839)
+
+If this executive proposal passes, then the LitePSM Keeper Network Job will be updated with the following contract calls and parameters as a consequence of the LITE-PSM-USDC-A DC-IAM and `buf` parameter changes described earlier in this document:
+
+- The old LitePSM job will be removed from the CronSequencer through contract call `sequencer.removejob(0x689cE517a4DfCf0C5eC466F2757D324fc292C8Be)`.
+- The new LitePSM job will be added to the CronSequencer through contract call `sequencer.addjob(0x0c86162ba3e507592fc8282b07cf18c7f902c401)`.
+- [`fill`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [rushThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be set to **20 million DAI**.
+- [`trim`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [gushThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be set to **20 million DAI**.
+- [`chug`](https://forum.makerdao.com/t/lite-psm-usdc-a-phase-1-test-period-proposed-parameters/24644#keeper-network-threshold-parameters-4): The [cutThreshold](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841#solution-4) will be remain unchanged at **300,000 DAI**.
+- The chainlog entry for [CRON_LITE_PSM_JOB](https://etherscan.io/address/0x689ce517a4dfcf0c5ec466f2757d324fc292c8be) will be updated to [0x0c86162ba3e507592fc8282b07cf18c7f902c401](https://etherscan.io/address/0x0c86162ba3e507592fc8282b07cf18c7f902c401).
+
+The new LitePSM job meets all the requirements outlined in the [LitePSM Job Post-Mortem](https://forum.makerdao.com/t/litepsm-job-post-mortem/24841).
+
+## Review
+
+Community debate on these topics can be found on the MakerDAO [Governance forum](https://forum.makerdao.com/). Please review any linked threads to inform your position before voting.
+
+---
+
+## Resources
+
```

Comment:
For more information on the LitePSM (LITE-PSM-USDC-A), please refer to the [LitePSM (LITE-PSM-USDC-A): Introduction and Overview](https://forum.makerdao.com/t/litepsm-lite-psm-usdc-a-introduction-and-overview/24512)